content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -59,3 +59,40 @@ Output
 | jenna      | black     | 35  | 60           |
 | chris      | smith     | 30  | 35           |
 | dave       | james     | 19  | 30           |
+
+## Example using partition by
```

Comment:
```suggestion
## Example 
```
Remove this `using partition by` from the example heading, and you can place it below the ##Example heading. 
For example, like this below:
##Example
###using partition by
//Example code